cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -566,9 +566,10 @@ func (uc *UpstreamController) queryService(stop chan struct{}) {
 					log.LOGGER.Warnf("message: %s process failure, response failed with error: %s", msg.GetID(), err)
 					continue
 				}
-				log.LOGGER.Warnf("message: %s process successfully", msg.GetID())
+				log.LOGGER.Infof("message: %s process successfully", msg.GetID())
 			default:
 				log.LOGGER.Infof("message: %s process failure, service operation: %s unsupported", msg.GetID(), msg.GetOperation())
+				continue
 			}
 			log.LOGGER.Infof("message: %s process successfully", msg.GetID())
```

Comment:
I think this will always be duplicate of 569 ? Can we remove this one ?

Reply:
ok

Reply:
Remove which one? 569 or 574? I think it is better to delete 569, because 569 is in a case of switch, we may add other cases in the future.

Reply:
I would suggest removing 573... for each case we can have a log message at the end which maybe different.

Reply:
There should be no difference.